vectordraw/templates/html/vectordraw.html

```diff
@@ -0,0 +1,130 @@
+{% load i18n %}
+<div class="vectordraw_block">
+
+  <h2>{{ self.display_name }}</h2>
+
+  {% if self.description %}
+    <div class="vectordraw-description">
+      {{ self.description|safe }}
```

Comment:
Why is the description considered HTML?  Shouldn't it be escaped like other things?